### DISCUSSION TITLE
Availability of the property `renderDay` property in v8?

### DISCUSSION THREAD

**Discussion by lmaze** [gpbl/react-day-picker#1174] [Support] [answered]
Hello @gpbl!

Version 8 of `react-day-picker` looks awesome ❤️
Is there any chance that the `renderDay` property available in version 7 will be ported to version 8? This feature was great but seems there is nothing about this in the version 8 roadmap 😢 Is that intended?

**Comment by gpbl** [comment, marked as answer]
@lmaze thanks for the heads up! 

What are you using the `renderDay` prop for? Using the `DayContent` custom component should be enough to replace `renderDay`: https://react-day-picker-next.netlify.app/guides/custom-components

```tsx
import * as React from 'react';
import { DayContent, DayContentProps, DayPicker } from 'react-day-picker';

import { format } from 'date-fns';

function DateTime(props: DayContentProps) {
  const dateTime = format(props.date, 'yyyy-MM-dd');
  return (
    <time dateTime={dateTime}>
      <DayContent {...props} />
    </time>
  );
}

export default function App() {
  return <DayPicker components={{ DayContent: DateTime }} />;
}
```

**Reply by lmaze** [reply]
@gpbl Sorry for the delay in my response, I missed your response. Thank you very much for responding so quickly and for the sample code.

This is exactly what I was looking for. I hadn't seen that it was possible to override the `DayContent` component with a custom component 😅 This is a very nice way to control the display of a day cell!

Thanks again for all the hard work on v8, can't wait for the final version!
